beyond-essence/inventory/toggle-switches.md

```diff
@@ -6,77 +6,63 @@ background_image: index.jpg
 
-The UI element "checkbox" is used quite similarly.
-Check the section "Choosing between toggle switch and checkbox", and have a look at the inventory entry [Checkboxes](/beyond-essence/inventory/checkboxes/) to find the right element for your use case.
+The UI element "checkbox" is used in our software quite similarly.
+Check the section "Choosing between toggle switches and checkboxes" below and have a look at the inventory entry [Checkboxes](/beyond-essence/inventory/checkboxes/) to find the right element for your use case.
 
 ## Use cases
 
-In the cockpit, toggle switches have two main use cases:
+In the cockpit, toggle switches have three main use cases:
 
-### Activate a feature
+### Activate significant features
 
-Some features need to be enabled by merchants before they can be used.
-In this context, a feature is defined as a functionality that has an impact on the whole workflow as additional processes are activated in the background.
+Some toggle switches in the cockpit have a significant impact on the functionality of the cockpit and the merchant's workflow.
 
-**Example: Activate stock level tracking**
+**Example: Website settings**
 
-* When activating the stock level tracking for a product, a new process is initiated.
-The stock is stored and then updated automatically or manually by the merchant.
-* The feature provides a new process, and therefore has a major impact on the merchant's workflow.
-* The feature might also be used on further pages in the cockpit.
+* **Website is live**: 
+When the merchant activates this toggle and saves the changes, the website goes live and will be visible to customers.
+Customers will be able to view and interact with the merchant's website.
+* **Allow visitors to make purchases on your website**: 
+When the merchant activates this toggle and saves the changes, a cart will be displayed at the top of the website and each product will have an "Add to cart" button, allowing visitors to order the merchant's products.
+
+💡 _Note: This is a temporary solution. The current implementation needs to be reviewed and updated in the future._
 
 ### Enhance cards with extensions
 
-The cockpit will be customizable.
+The cockpit is customizable.
 In order to allow merchants to adapt the cockpit to their needs, they can turn features on or off.
-This will be possible, for example, on single cards.
-
-**Example: Add the feature "product properties"**
-
-* If the feature is enabled, the merchant can add a list of product properties to use them e.g. in search filters.
-* If the feature is disabled, the merchant doesn't see this option, but can only use the description. For merchants with few and simple products this is sufficient, and the UI is less cluttered.
-
-The option is directly available.
-There is no need to confirm the action.
-
-{% image_custom image="/assets/img/pages/essence/switch-button_card_feature.png" width="50" align="center" %}
```

Comment:
Would you mind also removing the image with this PR?

Reply:
I think this is what the PR is doing... or do you mean remove it from the folder?